src/interactive/__snapshots__/accordionPanel.test.jsx.snap

```diff
@@ -314,6 +314,7 @@ exports[`AccordionPanel Panel with toggle switch exists and renders a toggle swi
                   <Button
                     aria-checked={false}
                     aria-labelledby="firstsection-label"
+                    buttonTag="button"
```

Comment:
I'd prefer we use something that we can standardize across components that accept html element customization...

`tag`?
`DOMElement`?
`htmlElement`?

Reply:
I'd vote for `htmlElement`, just because `htmlFor` is a thing

Reply:
But `htmlFor` is just a way to get around `for` being a reserved string in JavaScript, it's not describing the `for` in any way. If you don't think this makes it confusing to name this `htmlElement`, I'll continue with that.
Otherwise, how about `domElement`?